### PR TITLE
DISP-4084: Port work-hours Meteor package

### PR DESCRIPTION
**Pull request by aldeed**
Copied and adapted from https://github.com/DispatchMe/meteor-packages/tree/master/packages/work-hours

Code is the same other than switching to Promises and using SDK/API calls rather than MongoDB queries. See the README.md diff for syntax.

### CODE REVIEW

File: lib/entities/attachment.js [hunk 1]
```diff
@@ -1,25 +1,22 @@
 import { ATTACHMENTS } from '../endpoints';
```
Comment (aldeed) [inline lib/entities/attachment.js]: No real changes in this file. I just adjusted the arrow function syntax a bit and added a name for the exported function.

File: lib/entities/customer.js [hunk 2]
```diff
@@ -1,14 +1,11 @@
 import { CUSTOMERS } from '../endpoints';
```
Comment (aldeed) [inline lib/entities/customer.js]: No real changes in this file. I just adjusted the arrow function syntax a bit and added a name for the exported function.

File: lib/entities/job.js [hunk 3]
```diff
@@ -1,105 +1,102 @@
 import * as endpoints from '../endpoints';
```
Comment (aldeed) [inline lib/entities/job.js]: No real changes in this file. I just adjusted the arrow function syntax a bit and added a name for the exported function.

File: lib/fileHelpers.js [hunk 4]
```diff
@@ -4,7 +4,7 @@
  * @param  {String} dataURI
  * @return {Blob}
  */
-export const dataURItoBlob = function(dataURI) {
+function dataURItoBlob(dataURI) {
```
Comment (aldeed) [inline lib/fileHelpers.js]: Naming function to get rid of lint warnings

File: README.md [hunk 5]
```diff
@@ -157,3 +157,59 @@ client.get('/v1/jobs')
 .catch(err => console.error(err));
 ```
 
+## Organization-Scope Functions
+
+These functions are available on client.entities.organization(id).
+
+### addCustomer
```
Comment (aldeed) [inline README.md]: `addCustomer` was added previously so I'm just documenting it in this PR.

**Comment by davidjbeveridge** [comment]
👍 💯 🍰

**Comment by jraede** [comment]
Yep, LGTM. Pretty much just a straight port with the added support for fetching the org. 🐑

**Pull request merged**